readthedocs/telemetry/collectors.py

```diff
@@ -121,9 +134,60 @@ def _get_all_conda_packages(self):
             return packages
         return []
 
-    def _get_pip_packages(self, include_all=False):
+    def _get_user_pip_packages(self):
         """
-        Get all the packages installed by the user using pip.
+        Get all the packages to be installed defined by the user.
+
+        It parses all the requirements files specified in the config file by
+        the user (python.install.requirements) using ``dparse`` --a 3rd party
+        package.
+
+        If the version of the package is explicit (==) it saves that particular
+        version. Otherwise, if it's not defined, it saves ``undefined`` and if
+        it's a non deterministic operation (like >=, <= or ~=) it saves
+        ``unknown`` in the version.
```

Comment:
Can we have another status like "private" for private repositories? I don't fully understand the intention of collecting the data so I'll leave the suggestion here :)

Reply:
> Can we have another status like "private" for private repositories? 

I'm not sure to understand what "private" would mean here and how we would detect that. Can you expand?

> I don't fully understand the intention of collecting the data so I'll leave the suggestion here :)

This data is useful to answer questions like: "How many projects are using Sphinx 5.0.0?" and be able to differentiate that from "How many projects are _pinning_ Sphinx 5.0.0?"

Reply:
Using readthedocs.com one can build private git repos. I'm wondering if we can mark certain build data as private, rather than collecting it.